lib/db/mysql/index.js

```diff
@@ -177,6 +177,9 @@ const QUERY_CODE_DELETE_USER = 'DELETE FROM codes WHERE userId=?';
 const QUERY_DEVELOPER = 'SELECT * FROM developers WHERE email=?';
 const QUERY_DEVELOPER_DELETE = 'DELETE FROM developers WHERE email=?';
 const QUERY_PURGE_EXPIRED_TOKENS = 'DELETE FROM tokens WHERE clientId != UNHEX(?) AND expiresAt < NOW() LIMIT ?;';
+// TODO: better names and only active tokens, not all.
+const QUERY_ACTIVE_SERVICES_FOR_EMAIL = 'SELECT c.name, c.id, t.scope FROM fxa_oauth.clients AS c, fxa_oauth.tokens as t WHERE c.id = t.clientId AND email = ?';
+const QUERY_DELETE_TOKENS_BY_SERVICE = 'DELETE FROM tokens WHERE clientId=?';
```

Comment:
Since they are called `clients` in the codebase, I feel it would probably be less confusing to always call them `client`. So, `s/service/client/gi`?